spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -759,6 +759,51 @@ public void testPartitionedImportFromEmptyPartitionDoesNotThrow() {
         sql("SELECT * FROM %s ORDER BY id", tableName));
   }
 
+  @Test
+  public void testSkipOnError() throws IOException {
+    createUnpartitionedFileTable("parquet");
+
+    List<Object[]> source = sql("SELECT * FROM %s ORDER BY id", sourceTableName);
+    Assert.assertEquals(String.format("Rows in source table did not match\nExpected :%s rows \nFound    :%s",
```

Comment:
Usually we just have our expected stored as a List<Object[]> and compare against that. In this case I don't think you really need to check that the createUnpartitioned method worked correctly.

Reply:
removed.